### ISSUE TITLE
output WKT for JPS geometries

### ISSUE THREAD

**Issue by behrisch** [eclipse-sumo/sumo#14436]
the current debug code should be "upgraded" into two options, one defining the filename and one to specify whether geo-coordinates are wished.

**Comment by bcoueraud87** [comment]
Please assign me.

**Comment by bcoueraud87** [comment]
> the current debug code should be "upgraded" into two options, one defining the filename and one to specify whether geo-coordinates are wished.

We are talking about the main pedestrian network created by SUMO for JuPedSim.

**Comment by namdre** [comment]
@bcoueraud87 can this be closed?

**Comment by bcoueraud87** [comment]
@namdre Yes, [here](https://github.com/eclipse-sumo/sumo/pull/14656) is the corresponding pull request. The documentation has also been amended concerning options `--pedestrian.jupedsim.wkt` and `--pedestrian.jupedsim.wkt.geo`.
